setup-env/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Setup Environment'
+description: 'Setup dependencies for base container'
+
+runs:
+  using: "composite"

```

Comment:
What does "composite" mean here?

Reply:
Github Actions have different types: https://docs.github.com/en/actions/creating-actions/about-actions#types-of-actions
Basically, it means run on the parent workflow's environment without creating a new environment.